django-stubs/db/models/fields/__init__.pyi

```diff
@@ -219,6 +219,8 @@ class Field(RegisterLookupMixin, Generic[_ST, _GT]):
         limit_choices_to: Optional[_LimitChoicesTo] = ...,
         ordering: Sequence[str] = ...,
     ) -> _ChoicesList: ...
+    def _get_flatchoices(self) -> List[_Choice]: ...
+    flatchoices = property(_get_flatchoices)
```

Comment:
I am not sure that this is supported by mypy: `Revealed type is "Any"`

Try using `@property` as a decorator.

Repro:

```python
class A:
    def _get_flatchoices(self) -> int: ...
    flatchoices = property(_get_flatchoices)

reveal_type(A().flatchoices)
```

Reply:
Ah, well spotted!